services/kommander/0.7.0/kommander.yaml

```diff
@@ -17,7 +17,7 @@ spec:
         kind: HelmRepository
         name: mesosphere.github.io-kommander-charts
         namespace: kommander-flux
-      version: "${kommanderChartVersion:=v2.7.0-dev}"
+      version: "${kommanderChartVersion:=v2.7.0-dev.fluxbump}"
```

Comment:
TODO

Reply:
yep, will have to do it after 2 approvals and then admin merge

Reply:
done